### ISSUE TITLE
Implement intrinsicContentSize: 

### ISSUE THREAD

**Issue by jalone87** [psharanda/Atributika#103]
Maybe i am looking at the wrong approach (hints welcome), but wouldn't it make sense to implement `intrinsicContentSize:` so that the vertical layout constraint when using the AttributeLabel is not necessary (it will be created by the system and managed likewise) rather than creating it manually and having to update it after `sizeThatFits` and friends?

The AttributedLabel doesn't get along too well when embedded in a ScrollView as far as i can see with respect to a normal UILabel.

**Comment by psharanda** [comment]
Hi. Could you provide an example, I haven't noticed any issues with scrollview.

**Comment by jalone87** [comment]
If i have several multiline UILabel layed out vertically (in any container, UIView, ScrollView, etc) i will not have to set manually all the vertical height constraints not in InterfaceBuilder nor programmatically, since the layout system will be automatically creating ones for me using the intrinsicContentSize: of such labels (or using a placeholder in case of the InterfaceBuilder). With AttributedLabel on the other side i have to set manually all the height or vertical constraints to satisfy the layout equation.

**Comment by psharanda** [comment]
Yeah this definitely makes sense. Just released new version 4.9.5 where `AttributedLabel` got much better support in IB. So now there is no need to set placeholder size and it is possible to change most of props from IB. The only thing which still  needs to be set from code is `attributedText`.

**Comment by rivera-ernesto** [comment]
`intrinsicContentSize` requires to set a maximum width, otherwise the underlying `UITextView` will expand beyond screen bounds for long text lines.

I couldn't find an easy way to deduce this value from the view's "environment" but `layoutSubviews()` could be a good place.

For #119 I subclassed `AttributedLabel` to add  a `maxWidth` property:

```swift
class MaxWidthAttributedLabel: AttributedLabel
{
    var maxWidth: CGFloat!
    
    open override var intrinsicContentSize: CGSize
    {
        sizeThatFits(CGSize(width: maxWidth, height: .infinity))
    }
}
```
